sda-commons-client-jersey/src/test/java/org/sdase/commons/client/jersey/test/ClientTestApp.java

```diff
@@ -30,6 +31,7 @@ public static void main(String[] args) throws Exception {
   public void initialize(Bootstrap<ClientTestConfig> bootstrap) {
     bootstrap.getObjectMapper().disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES);
```

Comment:
Do you know why the test application uses custom configuration for the `ObjectMapper`?

Reply:
That's the OM for reading the config. Imho, we reuse the config for different test apps with different config classes and therefore have properties in the yaml that are not configurable in every test app. But actually, it seems to be the other app that's needs the tolerant reader. Should I try to remove the extra setting in this PR?

Reply:
Yeah. Let's see if the extra configuration is needed at all! 👍

Reply:
Removed: https://github.com/SDA-SE/sda-dropwizard-commons/pull/1680/commits/016003ee131237b4111d4972ca29f90b8f68d795